src/permissions.rs

```diff
@@ -150,6 +180,8 @@ permissions! {
         stdarch,
         team,
     }
+    crates_io_ops_apps {
+    }
```

Comment:
The apps are called `crates-io` and `staging-crates-io`. Since `-` is not a valid Rust identifier we'll need to call the permissions `crates_io` and `staging_crates_io` and replace `-` with `_` when the bot checks for permissions.